src/main/resources/crafter/engine/studio-server-config.properties

```diff
@@ -1,308 +1,325 @@
-# Disk repository root path
+# Defines repository type used for storing content. Possible values:
+#   default (disk repository)
+#   alfresco (alfresco)
+#   alfrescoext (alfresco with improved caching and performance)
+repositoryType=default
+# Defines security provider for accessing repository. Possible values:
+#   default (basic provider with 3 users - admin, author, approver)
+#   demo (users are stored in XML file)
+#   alfresco (alfresco users and groups are used for authentication)
+#   alfrescoext (alfresco users and groups are used for authentication)
+securityType=default
+# Path to the folder that is used as root for disk repository.
 repository.diskImplementation.path=./crafter/data/repo
-repository.gitImplementation.path=./crafter/data/gitrepository
-# Demo security provider config location
-crafter.studio.security.demo.configLocation=./crafter/data/gitrepository/global-configuration/security-config.xml
-# Alfresco repository url
+# Location of XML file that contains users for demo security provider.
+crafter.studio.security.demo.configLocation=${repository.diskImplementation.path}/security-config.xml
+# Base URL to use to access alfresco when configured alfresco repository and alfresco security provider.
 alfrescoUrl=http://127.0.0.1:8080/alfresco
-# Enable repository bootstrap
+# Turn content repository bootstrap on (value true) or off (value false).
 crafter.repository.bootstrapEnabled=true
-# Preview content root path for alfrescoext
+# Location where preview content is stored. It is used with alfrescoext repository type to improve performance.
 crafter.repository.previewRootPath=./crafter/data/repo
-# Rebuild Repositroy Metadata job batch size
+# Defines size of a batch when rebuilding crafter repository metadata stored in database.
 crafter.repository.rebuildMetadata.batchSize=100
 
 ##########################
 # Services Configuration #
 ##########################
-# Site configuration root path
-serviceConfig.configPath=/config/studio
-# Site configuration file name
+# Location where site configuration is stored in content repository.
+serviceConfig.configPath=/cstudio/config/sites/{site}
+# File name where site configuration is stored.
 serviceConfig.configFileName=site-config.xml
 
 ######################################
 # Dependencies Service configuration #
 ######################################
-# Ignore dependency rules for paths defined by comma separated regular expressions
+# Defines paths that will be excluded from dependency rules. Common usage is for files that are not site content themselves, but are system or files packaged with engine.
+# Valid values are regular expressions and for multiple regular expressions, they are separated by comma (,).
 dmDependenciesService.ignoreDependenciesRules=/templates/system/common/cstudio-support.*\\.ftl,/templates/web/navigation/.*\\.ftl
-# Enable manual dependency approving
+# Turn manual dependency approving on (value true) or off (value false) when publishing content.
```

Comment:
What does it mean to manually approve a dependency?
Why is it false?
What happens when I make it true?